tensorflow_serving/model_servers/load_model_prediction_service_impl.cc

```diff
@@ -0,0 +1,105 @@
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#include <string>
+
+#include "tensorflow_serving/model_servers/load_model_prediction_service_impl.h"
+#include "tensorflow_serving/config/model_server_config.pb.h"
+
+#include "grpc/grpc.h"
+#include "tensorflow_serving/model_servers/grpc_status_util.h"
+#include "tensorflow_serving/servables/tensorflow/get_model_metadata_impl.h"
+#include "tensorflow_serving/model_servers/model_platform_types.h"
+
+namespace tensorflow {
+namespace serving {
+
+namespace {
+
+int DeadlineToTimeoutMillis(const gpr_timespec deadline) {
+  return gpr_time_to_millis(
+      gpr_time_sub(gpr_convert_clock_type(deadline, GPR_CLOCK_MONOTONIC),
+                   gpr_now(GPR_CLOCK_MONOTONIC)));
+}
+
+ModelServerConfig BuildSingleModelConfig(const string& model_name,
+                                         const string& model_base_path) {
+  ModelServerConfig config;
+  LOG(INFO) << "Building single TensorFlow model file config: "
+            << " model_name: " << model_name
+            << " model_base_path: " << model_base_path;
+  tensorflow::serving::ModelConfig* single_model =
+      config.mutable_model_config_list()->add_config();
+  single_model->set_name(model_name);
+  single_model->set_base_path(model_base_path);
+  single_model->set_model_platform(
+      tensorflow::serving::kTensorFlowModelPlatform);
+  return config;
+}
+
+}  // namespace
+
+::grpc::Status LoadModelPredictionServiceImpl::LoadPredict(::grpc::ServerContext *context,
+                                              const PredictRequest *request,
+                                              PredictResponse *response) {
+  tensorflow::RunOptions run_options = tensorflow::RunOptions();
+  if (enforce_session_run_timeout_) {
+    run_options.set_timeout_in_ms(
+        DeadlineToTimeoutMillis(context->raw_deadline()));
+  }
+  ModelSpec m_spec = request->model_spec();
+  string m_name = m_spec.name();
+	LOG(INFO) << "Load Model " + m_name;
+
+  string m_path = "/data/models/" + m_name;
+  string model_platform = "tensorflow";
+  ModelServerConfig server_config = BuildSingleModelConfig(m_name, m_path);
+  core_->ReloadConfig(server_config);
+  //ModelConfigList mconfig = [
+  //  m_name,
+  //  m_path,
+  //  model_platform];
+  /*
+  const ModelConfigList list = server_config.model_config_list();
+  status = core_->ReloadConfig(server_config);
+  */
```

Comment:
Can we remove this comments for now please?

If this is required for next step, perhaps add a `TODO` comment to indicate the next intent?